includes/content-distribution/class-outgoing-post.php

```diff
@@ -189,4 +190,53 @@ function( $term ) {
 		}
 		return $data;
 	}
+
+	/**
+	 * Get post meta for distribution.
+	 *
+	 * @return array The post meta data.
+	 */
+	protected function get_post_meta() {
+		$reserved_keys = Content_Distribution::get_reserved_post_meta_keys();
+
+		$meta = get_post_meta( $this->post->ID );
+
+		if ( empty( $meta ) ) {
+			return [];
+		}
+
+		$meta = array_filter(
+			$meta,
+			function( $value, $key ) use ( $reserved_keys ) {
+				// Filter out reserved keys.
+				return ! in_array( $key, $reserved_keys, true );
+			},
+			ARRAY_FILTER_USE_BOTH
+		);
+
+		// Unserialize meta values and reformat the array.
+		$meta = array_reduce(
+			array_keys( $meta ),
+			function( $carry, $key ) use ( $meta ) {
+				$carry[ $key ] = array_map(
+					function( $v ) {
+						return maybe_unserialize( $v );
+					},
+					$meta[ $key ]
+				);
+				return $carry;
+			},
+			[]
+		);
+
+		/**
+		 * Filters the post meta data for distribution.
+		 *
+		 * @param array   $meta The post meta data.
+		 * @param WP_Post $post The post object.
+		 */
+		$meta = apply_filters( 'newspack_network_distributed_post_meta', $meta, $this->post );
```

Comment:
Same with an un-needed var here

Reply:
Same in 53de4f37d70e2beb1414cfb574d1bf2f3bf9000c